README.md

```diff
@@ -63,31 +63,35 @@ export default Ember.Component.extend({
 });
 ```
 
-# Installing
+### Installation
 
-```
-ember install ember-computed-style
```

Comment:
Please keep the installation instructions included in the readme

Reply:
Fixed